src/h5cpp/datatype/datatype.hpp

```diff
@@ -144,6 +145,20 @@ class DLL_EXPORT Datatype
   //!
   virtual void size(size_t size) const;
 
+  //!
+  //! @brief get sign of type in bytes
+  //!
+  //! @throws std::runtime_error in case of a failure
+  //!
+  virtual bool sign() const;
+
+  //!
+  //! @brief set the sign of a type in bytes
+  //!
+  //! @throws std::runtime_error in case of a failure
+  //!
+  virtual void sign(bool sign) const;
```

Comment:
Since we are using a boolean, and this, I agree, makes sense, perhaps to be linguisticlaly accurate these functions should be called `signed` rather than `sign`. That is the question they answer, so then the documentation might also reflect this.

Reply:
`signed` is c++ keyword so better not to use it.